README.md

```diff
@@ -2,48 +2,143 @@
 
 ![Ansible Lint](https://github.com/sap-linuxlab/community.sap_install/actions/workflows/ansible-lint.yml/badge.svg?branch=main)
 
-This Ansible Collection executes various SAP Software installations and configuration tasks for running various SAP solutions and deployment scenarios on Linux operating systems (RHEL or SLES).
+## Description
 
-This includes handlers for SAP HANA database lifecycle manager (HDBLCM) and SAP Software Provisioning Manager (SWPM), and can be combined with other Ansible Collections to provide end-to-end automation _(e.g. provision, download, install, operations)_.
+This Ansible Collection executes various SAP Software installations and configuration tasks for various SAP solutions and deployment scenarios on supported Linux operating systems.
 
+Included roles cover range of tasks:
+- Preparation of Operating system and SAP installation media before installation
+- Installation of SAP Database, either SAP HANA or Oracle Database
+- Installation of SAP Products, like SAP S4HANA, SAP BW4HANA, SAP Netweaver, SAP Solution Manager and others.
+- Configuration of replication of SAP HANA and High Availability clusters for SAP HANA and SAP Netweaver
 
-**Examples of verified installations include:**
 
+## Requirements
+
+### Control Nodes
+Operating system:
+- Any operating system with required Python and Ansible versions.
+
+Python: 3.11 or higher
+
+Ansible: 9.9.x
+
+Ansible-core: 2.16.x
```

Comment:
ansible-core 2.17.4 requires python >=3.10, see 

https://pypi.org/project/ansible-core/2.17.4/

Above it says min required python is 3.11.

What higher python version do you mean?

Reply:
ansible-core 2.17 is not recommended as agreed in governance meeting.
Their breaking change in 2.17 requires Python 3.7 or 3.9 (no idea which because their own explanations mention both) - [SyntaxError: future feature annotations is not defined](https://github.com/ansible/ansible/issues/82068)

We also agreed to use latest or n-1 for Python and Ansible that we support, which in this case is:
ansible-core 2.16.x
Python 3.11 and 3.12